platformio.ini

```diff
@@ -16,59 +16,53 @@ test_dir = firmware/test
 
 [env]
 platform = platformio/ststm32@^17.3.0
-
-# WARNING: For whatever reason, this CANNOT be the
```

Comment:
Re-add comments - they were removed by platformio's automatic install modifications of this